rogue-thi-app/pages/calendar.js

```diff
@@ -178,25 +180,43 @@ export default function Calendar () {
             <ListGroup variant="flush">
               <ReactPlaceholder type="text" rows={10} ready={events}>
                 {events && events.length === 0 && (
-                  <ListGroup.Item className={styles.item}>
-                    Es sind derzeit keine Veranstaltungstermine verfügbar.
-                  </ListGroup.Item>
+                    <ListGroup.Item className={styles.item}>
+                      Es sind derzeit keine Veranstaltungstermine verfügbar.
+                    </ListGroup.Item>
```

Comment:
It looks like your IDE is set to indent with 4 spaces instead of 2 which results in this file being sometimes 2 and sometimes 4 spaces indented in your code.